python/lsst/faro/base/CatalogMeasurementBase.py

```diff
@@ -101,7 +103,14 @@ def run(self, **kwargs):
         return self.measure.run(self.config.connections.metric, **kwargs)
 
     def _getReferenceCatalog(self, butlerQC, dataIds, refCats, filterList, epoch=None):
-        """Load reference catalog in sky region of interest.
+        """Load reference catalog in sky region of interest and optionally applies proper
+        motion correction and color terms.
+
+        Loads the `lsst.afw.table.SimpleCatalog` reference catalog, computes ra and dec (optionally)
+        applying a proper motion correction and reference magnitudes transformed to the lsst bandpass.
+
+        returns a refCat with both the original loaded reference catalog and
+        the coorected coordinates (ra,dec) and reference magnitudes (refMag-/refMagErr)
```

Comment:
Suggest `(refMag-/refMagErr-)` and also below just for consistency.

The doc string might want to use the words "color terms" when referring to the reference magnitudes to make clear that the fluxes of reference objects have been converted to the photometric system of the data that is being analyzed.

Reply:
Updated:
> Loads the `lsst.afw.table.SimpleCatalog` reference catalog, computes ra and dec
> (optionally) applying a proper motion correction. Also, color terms
> are (optionally) applied to the reference magnitudes in order to transform
> them to the data's photometric system.
>
>  returns a refCat with both the original loaded reference catalog and
>  the coorected coordinates (ra,dec) and transformed reference magnitudes
>  (refMag-/refMagErr-)